src/jenkins.py

```diff
@@ -824,13 +825,22 @@ def _set_jenkins_system_message(message: str, container: ops.Container) -> None:
     Args:
         message: The system message to display.
         container: The Jenkins workload container.
+
+    Raises:
+        ValidationError: if invalid JCasC file was encountered.
     """
     jcasc_yaml = container.pull(JCASC_CONFIG_FILE_PATH, encoding="utf-8").read()
     config = yaml.safe_load(jcasc_yaml)
-    config["jenkins"]["systemMessage"] = message
-    container.push(
-        JCASC_CONFIG_FILE_PATH, yaml.dump(config), encoding="utf-8", user=USER, group=GROUP
-    )
+    try:
+        config["jenkins"]["systemMessage"] = message
+        container.push(
```

Comment:
Should this be outside the `try...except`?

Reply:
Fixed, thank you!